src/freenet/client/filter/HTMLFilter.java

```diff
@@ -2140,6 +2142,25 @@ static class TagVerifier {
 					this.booleanAttrs.add(booleanAttrs[x]);
 				}
 			}
+			// https://w3c.github.io/aria/
+			this.allowedRole = new HashSet<String>(Arrays.asList("alert","alertdialog","application","article",
+				"banner","blockquote","button",
+				"caption","cell","checkbox","code","columnheader","combobox","command","comment","complementary","composite","contentinfo",
+				"definition","deletion","dialog","directory","document",
+				"emphasis",
+				"feed","figure","form",
+				"generic","grid","gridcell","group",
+				"heading",
+				"image","img","input","insertion",
+				"landmark","link","list","listbox","listitem","log",
+				"main","mark","marquee","math","menu","menubar","menuitem","menuitemcheckbox","menuitemradio","meter",
+				"navigation","none","note",
+				"option",
+				"paragraph","presentation","progressbar",
+				"radio","radiogroup","range","region","roletype","row","rowgroup","rowheader",
+				"scrollbar","search","searchbox","section","sectionhead","select","separator","slider","spinbutton","status","strong","structure","subscript","suggestion","superscript","switch",
+				"tab","table","tablist","tabpanel","term","textbox","time","timer","toolbar","tooltip","tree","treegrid","treeitem",
+				"widget","window"));
```

Comment:
Functionally this looks fine.

As for formatting, I would prefer each value to be on a separate line, giving the list a more visually consistent appearance. A similar approach is taken for the long lists around line 2020.